packages/gafl-webapp-service/src/processors/__tests__/date-and-time-display.spec.js

```diff
@@ -34,6 +43,39 @@ describe('displayStartTime', () => {
     expect(moment.utc).toHaveBeenCalledWith(startDate, null, expect.any(String))
   })
 
+  it.each([[50], [10000], [0]])(
+    'expected text replaces the date string if start after payment is selected using correct constant',
+    async startAfterMinutes => {
+      constant.START_AFTER_PAYMENT_MINUTES = startAfterMinutes
+      const startDate = '2021-01-01T00:00:00.000Z'
+      const mssgs = getSampleRequest().i18n.getCatalog()
+      const expectedReturnText = constant.START_AFTER_PAYMENT_MINUTES + mssgs.licence_summary_minutes_after_payment
```

Comment:
It's preferable to use template quotes rather than the concatenation operator, e.g.
```
const expectedReturnText = `${constant.START_AFTER_PAYMENT_MINUTES}${mssgs.licence_summary_minutes_after_payment}
```
Also, for clarity, use the passed-in variable rather than what you've assigned it to:
```
const expectedReturnText = `${startAfterMinutes}${mssgs.licence_summary_minutes_after_payment}
```
Better still, hard-code the expected return text in a case variable, like in "displays the date in the correct format, when permission date is %s and current date and time is %s" below. Avoid too much logic in tests, otherwise you'll get to a point you need a test for your test!